### DISCUSSION TITLE
Data Scientist who can translate

### DISCUSSION THREAD

**Discussion by shadyshadi** [WeAreMahsaAmini/Core#80] [Introductions]
Hi I'm Shadi! I'm a data scientist and can help with data analysis, coding (Python) and translations from Farsi to English and vice versa.

**Comment by syavash** [comment]
Welcome, @shadyshadi. We are going to listen to Twitter firehose for the relevant hashtags. This means we will have a list of all tweets shared with these hashtags.

Based on your skills, what are some of the things you believe we can extract from this data, if any?

**Reply by shadyshadi** [reply]
I can do Natural Language Processing if we want to extract specific information. For instance, to track news of attacks, gatherings from Iran or outside. Depending on the type of accompanying data on the tweets there are things to do for geo locational mapping. If we set an objective or multiple I can help map out practical possibilities.
